dist-persist/wbstack/src/Settings/LocalSettings.php

```diff
@@ -643,8 +643,14 @@ function onPersonalUrlsConfirmAccount( array &$personal_urls, Title $title, Skin
 
     $wgSearchType = 'CirrusSearch';
     $wgCirrusSearchDefaultCluster = 'default';
+    
     // T308115
-    $wgCirrusSearchShardCount = [ 'archive' => 1, 'content' => 1, 'general' => 1, 'titlesuggest' => 1 ];
+    $wgCirrusSearchShardCount = [ 'content' => 1, 'general' => 1 ];
+
+    // T309379
+    $wgCirrusSearchEnableArchive = false;
+    $wgCirrusSearchPrivateClusters = [ 'non-existing-cluster' ];
```

Comment:
This doesn't like just an empty array here for some reason.

Reply:
cool, I dug into this a bit but didn't come to any conclusion. For now let's just move forwards with this